eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/runner/AlertBolt.java

```diff
@@ -81,6 +81,7 @@ public void execute(Tuple input) {
         this.streamContext.counter().scope("execute_count").incr();
         try {
             PartitionedEvent pe = deserialize(input.getValueByField(AlertConstants.FIELD_0));
+            LOG.info("Alert bolt {} received event: {}", boltId, pe.getEvent());
```

Comment:
When the throughput increases, this log (printing log per event) may be the bottle neck.